_2020/data-wrangling.md

```diff
@@ -286,7 +291,8 @@ field of the line should be equal to 1 (that's the count from `uniq
 expression. And the block just says to print the username. We then count
 the number of lines in the output with `wc -l`.
 
-However, `awk` is a programming language, remember?
+However, `awk` is a programming language, remember? So we can, for example, 
+define and update variables.
```

Comment:
I don't think this added sentence adds much.

Reply:
Let's leave it out then.